contrib/nsqadmin.cfg.example

```diff
@@ -1,72 +0,0 @@
-## log verbosity level: debug, info, warn, error, or fatal
```

Comment:
Why samples are removed?

Reply:
Just in case `TestConfigFlagParsing` tests will be broken after removing those files...